### PR TITLE
mobile-menu -section

### PR DESCRIPTION
**Pull request by sj1978**
In this section we : 
.Created a new section for the mobile menu
.Added styles to the menu
.Created a javascript file.
.Added functionality to the menu with javascript

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,8 +58,16 @@
         /></a>
       </nav>
     </header>
+    <section class="mobile-menu">
+      <a href="#" class="close-btn" id="close-menu-btn">X</a>
+      <ul class="a-color">
+         <li><h3>Portfolio</h3><a class="listBtns" href="#profile">></a></li>
+         <li><h3>About</h3><a class="listBtns"  href="#about">></a></li>
+         <li><h3>Contact</h3><a class="listBtns"  href="#contact">></a></li>
```
Comment (GzimAsani) [inline index.html:66]: - [ ] When the user clicks either of those buttons, a correct part of the page should be displayed
Screenshot from 2021-10-05 00-40-38
d

Comment (GzimAsani) [inline index.html:66]: - [ ] Please  remove those `>` 
Screenshot from 2021-10-05 00-40-38
Screenshot from 2021-10-05 00-40-38

e

Comment (GzimAsani) [inline index.html:66]: - [ ] When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.

File: .github/workflows/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
```
Comment (GzimAsani) [inline .github/workflows/.eslintrc.json:25]: This folder should be at the root of your project but currently, it is in /.github/.workflows/.eslintrc.json

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @sj1978  
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good use of gitflow :+1: 

## Required Changes ♻️
- [ ] _Check the comments under the review for more information._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/.eslintrc.json (hunk 2 above)
Comment (od-c0d3r) [reply .github/workflows/.eslintrc.json]: The requirement has been fulfilled 👏🏼

File: index.html (hunk 1 above)
Comment (od-c0d3r) [reply index.html]: The requirement has been fulfilled 👏🏼

Comment (od-c0d3r) [reply index.html]: The requirement has been fulfilled 👏🏼

Comment (od-c0d3r) [reply index.html]: The requirement has been fulfilled 👏🏼

**Review by od-c0d3r — APPROVED** [review]
Hi @sj1978, @Hope1226 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**